packages/create-llama/templates/components/sample-projects/llamapack/README-template.md

```diff
@@ -2,14 +2,19 @@
 
 ## Quickstart
 
-1. Check above instructions for setting up your environment and export required environment variables
-   For example, if you are using bash, you can run the following command to set up OpenAI API key
+1. Check above instructions for setting up your environment. Your project will need to install `[llama_pack_package_name]` package. You can install it using pip
+
+```bash
+pip install [llama_pack_package_name]
```

Comment:
can we skip this step by modifying the generated pyproject.toml file? by adding the necessary dependency?